tutorial/README.md

```diff
@@ -1,17 +1,20 @@
-## Introduction 
+## Introduction
 
 This brief, self-contained tutorial will walk the user through an end-to-end run of `mama`. We encourage new users to consult this document prior to using the package on their own data. We provide individual-level data of 10 people from 1000 Genomes (5 from EUR sample, 5 from EAS sample) and ~800 randomly selected
 SNPs on chromosome 22. We also give a sample EAS GWAS of BMI (CKB + BBJ) and EUR GWAS of BMI (UKB) restricted to chromosome 22.
 
+Note: Due to the low number of SNPs in the LDSC reference panel, the MAMA output of this tutorial will be very sparse -- only 45 SNPs after QC and restricting

```

Comment:
In case anything changes with the tutorial data or we make any improvements or updates to the code (perhaps adding new filters or removing existing ones for some reason?) that alters this number, it might be better to not give a precise number.  Saying something like \"<50\" would mean we don't have to remember to come update this as often (still might need to occasionally)

Reply:
Makes sense.